Makefile

```diff
@@ -62,10 +62,11 @@ endef
 help:
 	$(info $(HELP_TEXT))
 
-deps:
-	go get -u github.com/golang/dep/...
-	go get -u github.com/golang/lint/golint
-	dep ensure -vendor-only -v
+gomodcheck:
+	@go help mod > /dev/null || (@echo gorilla requires Go version 1.11 or higher && exit 1)
+
+deps: gomodcheck
+	@go mod download
```

Comment:
I don't think `go mod download` is necessary. Dependencies will be retrieved any time `go build` or `go test` is ran. `deps` can likely be removed all together.

Instead, maybe we should call `gomodcheck` as the first step of `make build` and `make test`, so we exit if `go mod` is not supported.

Reply:
Not positive if I did this as requested, but in commit 6c9b9b3 I threw a `gomodcheck` into the `prebuild` command... directive?... block?... 🤷‍♂️ And removed altogether the `deps:` command... directive?... block?...